File: isoft_android/linkknown/app/src/main/java/com/linkknown/ilearning/model/BaseResponse.java
package com.linkknown.ilearning.model;

import java.io.Serializable;

import lombok.Data;

@Data
public class BaseResponse implements Serializable {

    // 状态
    private String status;
    private String errorMsg;

    public boolean isSuccess () {
        return "SUCCESS".equals(status);
    }

}

File: isoft_flutter/linkknown/lib/response/base_response.dart

class BaseResponse {
  String status;
  String errorMsg;

  BaseResponse();

  BaseResponse.fromJson(Map<String, dynamic> json) {
    status = json['status'];
    errorMsg = json['errorMsg'];
  }

  bool isSuccess () {
    return status == "SUCCESS";
  }
}
